src/components/installedAppsView/InstalledAppsView.js

```diff
@@ -17,11 +17,31 @@ class InstalledAppsView extends React.Component {
   }
 
   static getStatusForApp(app) {
-    const provisioningStatus = 'Provisioning';
-    if (app.status && app.status.conditions && app.status.conditions[0]) {
-      return app.status.conditions[0].status === 'True' ? 'Provisioned' : provisioningStatus;
+    let statusIcon;
+    let statusStyle;
+    let provisioningStatus = app.status.provisionStatus || 'Provisioning';
+
+    statusIcon = provisioningStatus === 'Provisioned' ? 'pficon-ok' : 'spinner spinner-xs spinner-inline';
+    statusStyle = provisioningStatus === 'Provisioned' ? { color: '#3f9c35' } : { color: '#4d5258' };
+
+    if (provisioningStatus === 'NotProvisioned') {
+      provisioningStatus = 'Provision Failed';
+      statusIcon = 'pficon-error-circle-o';
+      statusStyle = { color: '#cc0000' };
```

Comment:
I tried using the patternfly color palette here like `{ color: '$pf-red' } but I couldn't get it working.

Reply:
The correct variable is `$color-pf-red-100`.